### DISCUSSION TITLE
full case stl

### DISCUSSION THREAD

**Discussion by InvaderZimm** [sanni/cartreader#344] [Need help]
for some reason i cant download the full case stl files none of them i just get the raw code

**Comment by InvaderZimm** [comment]
i was trying to download the bottom, top or buttons and none of them will let you do so

**Comment by sanni** [comment]
![easgfqgegqgq](https://user-images.githubusercontent.com/19169494/158081104-422d73d1-2e48-4493-92e3-b3120593d55f.jpg)

**Reply by InvaderZimm** [reply]
wow i feel so dumb thank you again!

**Reply by iceyman** [reply]
where do i find the stl files for a full case for version 4

**Reply by sanni** [reply]
Version 4 does not have a full case yet.
